File: PicturesApi/PooPosting.Api/Models/LikeResult.cs
using PooPosting.Api.Enums;
using PooPosting.Api.Models.Dtos.Like;
using PooPosting.Api.Models.Dtos;

namespace PooPosting.Api.Models;

public class LikeResult
{
    public LikeState LikeState { get; set; }
    public IEnumerable<LikeDto> Likes { get; set; }
    public int LikeCount { get; set; }
    public int DislikeCount { get; set; }
}

File: PooPosting/PooPosting.View/src/app/Models/ApiModels/Patch/LikeResult.ts
import {LikeModel} from "../Get/LikeModel";
import {LikeState} from "../../../Enums/LikeState";

export interface LikeResult {
  likeCount: number,
  dislikeCount: number,
  likes: LikeModel[],
  likeState: LikeState
}
